sys/shell/commands/sc_openwsn.c

```diff
@@ -0,0 +1,569 @@
+                    return -1;
+                }
+            }
+            else {
+                puts("Error: a neighbor address must be supplied");
+                return -1;
+            }
+        }
+        int ret = sixtop_request(IANA_6TOP_CMD_CLEAR, &neigh, 0, 0, NULL,
+                                 NULL, IANA_6TISCH_SFID_MSF, 0, 0);
+        if (ret) {
+            puts("Error: 6top request failed");
+            return -1;
+        }
+    }
+
+    if (!strcmp(argv[1], "add") || !strcmp(argv[1], "rmv") || \
+        !strcmp(argv[1], "rel")) {
+        return _6top_manage_cmd(argc, argv);
+    }
+
+    _print_6top_usage();
+    return 1;
+}
+
+static void _print_queue_usage(void)
+{
+    puts("Usage:");
+    puts("\tqueue list: show all Openqueue entries");
+    puts("\tqueue rmv <creator>: remove all entries from <creator> in queue");
+}
+
+static int _queue_cmd(int argc, char **argv)
+{
+    if (argc < 2) {
+        _print_queue_usage();
+        return 1;
+    }
+
+    if (!strcmp(argv[1], "list")) {
+        bool empty_queue = true;
+
+        for (uint8_t i = 0; i < QUEUELENGTH; i++) {
+            if (openqueue_vars.queue[i].creator ||
+                openqueue_vars.queue[i].owner) {
+                empty_queue = false;
+                uint8_t creator = openqueue_vars.queue[i].creator;
+                uint8_t owner = openqueue_vars.queue[i].owner;
+                printf("Creator: %.9s [%d], ", _get_component(creator),
+                       creator);
+                printf("Owner: %.9s [%d]\n", _get_component(owner), owner);
+            }
+        }
+        if (empty_queue) {
+            puts("Queue is empty");
+        }
+        return 0;
+    }
+
+    if (!strcmp(argv[1], "rmv")) {
+        if (argc != 3) {
+            _print_queue_usage();
+            return -1;
+        }
+
+        uint8_t creator = atoi(argv[2]);
+
+        if (creator == 0) {
+            puts("Error: invalid input value\n");
+            return 1;
+        }
+        else {
+            printf("Removing entries created by: %.9s [%d]\n", \
+                   _get_component(creator), creator);
+            openqueue_removeAllCreatedBy(creator);
+            return 0;
+        }
+    }
+
+    _print_queue_usage();
+    return 1;
+}
+
+static int _scheduler_cmd(char *arg)
+{
+    (void)arg;
+
+    printf("Current tasks:%i\n", scheduler_dbg.numTasksCur);
+    printf("Max tasks:    %i\n", scheduler_dbg.numTasksMax);
+
+    return 0;
+}
+
+static void _print_usage(void)
+{
+    puts("Usage:");
+    puts("\topenwsn neigh: show neighbor table");
+    puts("\topenwsn queue: Openqueue management commands");
+    puts("\topenwsn cell: cell management commands (directly on schedule)");
+    puts("\topenwsn 6top: 6top request commands");
+    puts("\topenwsn sched: show openos scheduler information");
+}
+
+int _openwsn_handler(int argc, char **argv)
+{
+    if (argc < 2) {
+        _print_usage();
+        return 1;
```

Comment:
```suggestion
        return -1;
```
And there's a last one at the end of this function